### PR TITLE
forms updated

### PR DESCRIPTION
**Pull request by Sebcoppo**
hello ! 
here is my PR
Family and senior forms updated
hope to enjoy

### CODE REVIEW

File: app/views/devise/registrations/_new_family.html.erb [hunk 1]
```diff
@@ -27,21 +27,73 @@
...
+
+            <%= f.input :link,
+                        label: "Quel est votre lien avec le bénéficiaire?",
                         required: true,
-                        input_html: { autocomplete: "address", :id => "family_address", class: "seko-search-bar" }  %>
-            <%= f.input :address,
-                        label: "Complément d'adresse",
+                        collection: [
+                          ['Parent proche', 'Parent Proche'],
+                          ['Ami(e)', 'Ami(e)'],
```
Comment (MaximeAlexandre) [inline app/views/devise/registrations/_new_family.html.erb:36]: Carefull, in a collection, in each array, the first element will be the value to store in the DB and the second will be the text display to the user. So no spaces or upcase for the first element.
["close_parent", "Parent Proche"] or ["0", "Parent Proche"]
Same for the second array with "Amie".

File: app/views/devise/registrations/_new_family.html.erb [hunk 2]
```diff
@@ -27,21 +27,73 @@
...
+                        placeholder: "Quels sont ses passe-temps? Combien a t il de petits enfants?",
                         required: false,
                         input_html: { class: "seko-search-bar" } %>
+            <%= f.input :pathology,
+                        label: "Souffre t il d'une pathologie chronique?",
+                        collection: [
+                          ['Parkinson', 'Parkinson' ],
+                          ['Alzheimer', 'Alzheimer'],
+                          ['Autre', 'Autre']
+                        ],
```
Comment (MaximeAlexandre) [inline app/views/devise/registrations/_new_family.html.erb:76]: See last comment above.

File: app/views/devise/registrations/_new_senior.html.erb [hunk 3]
```diff
@@ -48,22 +48,32 @@
                         input_html: { class: "seko-search-bar" } %>
             <%= f.input :pathology,
                         label: "Souffrez vous d'une pathologie chronique?",
-                        collection: ['Parkinson','Alzheimer', 'Autre'],
+                        collection: [
+                          ['Parkinson', 'Parkinson' ],
+                          ['Alzheimer', 'Alzheimer'],
+                          ['Autre', 'Autre']
+                        ],
```
Comment (MaximeAlexandre) [inline app/views/devise/registrations/_new_senior.html.erb:55]: See comment in _new_family.html.erb about collection

File: app/views/devise/registrations/_new_senior.html.erb [hunk 4]
```diff
@@ -48,22 +48,32 @@
...
                         required: false,
                         input_html: { class: "seko-search-bar" } %>
-
-
         </div>
         <div class="form-actions seko-flex">
-          <%= f.button :submit, "S'inscrire", class: "btn seko-button w-50 m-3" %>
-          <%= render "devise/shared/links" %>
+          <%= f.button :submit, "S'inscrire", class: "btn seko-button w-100 m-3" %>
+          <!-- <%= render "devise/shared/links" %> -->
```
Comment (MaximeAlexandre) [inline app/views/devise/registrations/_new_senior.html.erb:76]: Comment line to delete or uncomment ?

**Review by MaximeAlexandre — CHANGES_REQUESTED** [review]

File: app/views/devise/registrations/_helper_form.erb [hunk 5]
```diff
@@ -67,7 +67,7 @@
               input_html: { class: "seko-search-bar" } %>
   </div>
   <div class="form-actions seko-flex">
-    <%= f.button :submit, "S'inscrire", class: "btn seko-button w-50 m-3" %>
-    <%= render "devise/shared/links" %>
+    <%= f.button :submit, "S'inscrire", class: "btn seko-button w-100 m-3" %>
+   <!--  <%= render "devise/shared/links" %> -->
```
[the remark this answers is not in the document]
Comment (Sebcoppo) [reply app/views/devise/registrations/_helper_form.erb]: I delete it !

File: app/views/devise/registrations/_new_family.html.erb (hunk 1 above)
Comment (Sebcoppo) [reply app/views/devise/registrations/_new_family.html.erb]: I've updated arrays

File: app/views/devise/registrations/_new_family.html.erb (hunk 2 above)
Comment (Sebcoppo) [reply app/views/devise/registrations/_new_family.html.erb:76]: I've updated arrays.

File: app/views/devise/registrations/_new_family.html.erb [hunk 6]
```diff
@@ -27,21 +27,73 @@
...
+                         collection: ['Oui','Non'],
+                         include_blank: true,
+                         required: false,
+                         input_html: { class: "seko-search-bar" } %>
         </div>
         <div class="form-actions seko-flex">
-          <%= f.button :submit, "S'inscrire", class: "btn seko-button w-50 m-3" %>
-          <%= render "devise/shared/links" %>
+          <%= f.button :submit, "S'inscrire", class: "btn seko-button w-100 m-3" %>
+         <!--  <%= render "devise/shared/links" %> -->
```
[the remark this answers is not in the document]
Comment (Sebcoppo) [reply app/views/devise/registrations/_new_family.html.erb]: I've deleted the line

File: app/views/devise/registrations/_new_senior.html.erb (hunk 3 above)
Comment (Sebcoppo) [reply app/views/devise/registrations/_new_senior.html.erb:55]: Arrays updated !

File: app/views/devise/registrations/_new_senior.html.erb (hunk 4 above)
Comment (Sebcoppo) [reply app/views/devise/registrations/_new_senior.html.erb]: Line deleted !

**Review by MaximeAlexandre — APPROVED** [review]

**Pull request merged**